### PR TITLE
Clean class.jetpack.php

### PR DESCRIPTION
**Pull request by dereksmart**
Removes unused code from class.jetpack.php.  Notably: 

- old admin_ajax callbacks used in the legacy admin
- ID crisis resolves

### CODE REVIEW

File: class.jetpack.php [hunk 1]
```diff
@@ -486,15 +486,9 @@ private function __construct() {
 		// If any module option is updated before Jump Start is dismissed, hide Jump Start.
 		add_action( 'update_option', array( $this, 'jumpstart_has_updated_module_option' ) );
 
-		// Identity Crisis AJAX callback function
-		add_action( 'wp_ajax_jetpack_resolve_identity_crisis', array( $this, 'resolve_identity_crisis_ajax_callback' ) );
-
 		// JITM AJAX callback function
 		add_action( 'wp_ajax_jitm_ajax',  array( $this, 'jetpack_jitm_ajax_callback' ) );
 
-		add_action( 'wp_ajax_jetpack_admin_ajax',          array( $this, 'jetpack_admin_ajax_callback' ) );
```
Comment (eliorivero) [inline class.jetpack.php]: @dereksmart there's a reference to this `jetpack_admin_ajax` here https://github.com/Automattic/jetpack/blob/a1fb89c9206fa12e6af8c8c43af1164d2086d8c7/_inc/jp.js#L251

File: class.jetpack.php [hunk 2]
```diff
@@ -486,15 +486,9 @@ private function __construct() {
...
-		add_action( 'wp_ajax_jetpack_admin_ajax_refresh',  array( $this, 'jetpack_admin_ajax_refresh_data' ) );
```
Comment (eliorivero) [inline class.jetpack.php]: reference to `jetpack_admin_ajax_refresh` here https://github.com/Automattic/jetpack/blob/a1fb89c9206fa12e6af8c8c43af1164d2086d8c7/_inc/jp.js#L317

Comment (dereksmart) [inline class.jetpack.php]: That will be removed in #4697

**Comment by zinigor** [comment]
Nice! LGTM!

**Pull request merged**